lib/ts/Arena.h

```diff
@@ -166,4 +167,34 @@ Arena::str_store(const char *str, size_t len)
   return mem;
 }
 
-#endif /* __ARENA_H__ */
+struct JemallocArena : public std::allocator<uint64_t>
```

Comment:
I was under the impression that ATS Arenas and jemalloc Arenas were two different concepts. Is this not the case?